lightning/src/chain/package.rs

```diff
@@ -551,6 +551,32 @@ impl PackageSolvingData {
 			_ => { mem::discriminant(self) == mem::discriminant(&input) }
 		}
 	}
+	fn as_tx_input(&self, previous_output: BitcoinOutPoint) -> TxIn {
+		let sequence = match self {
+			PackageSolvingData::RevokedOutput(_) => Sequence::ENABLE_RBF_NO_LOCKTIME,
+			PackageSolvingData::RevokedHTLCOutput(_) => Sequence::ENABLE_RBF_NO_LOCKTIME,
```

Comment:
I think we’re testing the  “anchor channel”-enabled revoked counterparty commitment offered HTLC outputs case in `test_anchors_aggregated_revoked_htlc_tx` though I’m unsure we’re testing the claim with justice transaction on revoked counterparty commitment received HTLC ? All routed payments L2029 are in the same direction.